src/target/tx/devo/devo10/Makefile

```diff
@@ -8,4 +8,7 @@ LANGUAGE   := devo10
 
 OPTIMIZE_DFU := 1
 
+TARGET=devo10
+
```

Comment:
Why do I need this in every Makefile?  Isn't it inferrable by the directory I'm in?  The less things I need to define the better.
On the other hand, hiding away definitions makes things harder to read...I'm torn

Reply:
I struggled as well.
I can pass TARGET from Makefile, maybe I should do that.